### ISSUE TITLE
Monkey Patching of rendering functions

### ISSUE THREAD

**Issue by shyamd** [materialsproject/crystaltoolkit#76]
It appears the conversion of objects to scenes is monkey patched into place. There should be a better way to do this.

**Comment by mkhorton** [comment]
Thinking of moving it to pymatgen. Monkey patched right now to facilitate
an easy move

**Comment by mkhorton** [comment]
Specifically the get_scene() method for anything that can be displayed in
3D...

**Comment by mkhorton** [comment]
Any thoughts on this btw?

**Comment by shyamd** [comment]
Yeah, i don't like the idea of putting the get_scene into pymatgen since that makes a bi-directional dependency. 

My first thoughts are to make an Abstract `Renderable` class that also implements a registry pattern. Then we can make adaptors that implement `Renderable` and register themselves so that we can just something like `Renderable.render(object)` without having to know what it is.

**Comment by mkhorton** [comment]
Yeah, I wasn’t intending to make crystal toolkit the dependency for that
reason. It’d require moving the scene classes etc into a small standalone
package I think.

The abstract Renderable is interesting
